nextcord/file.py

```diff
@@ -144,6 +167,7 @@ def reset(self, *, seek: Union[int, bool] = True) -> None:
             self.fp.seek(self._original_pos)
 
     def close(self) -> None:
-        self.fp.close = self._closer
-        if self._owner:
+        if self._owner or self.force_close:
             self._closer()
+        else:
+            self.fp.close = self._closer
```

Comment:
`self.fp.close = self._closer` should probably stay where it is. Keeping the order the same and just adding `or self.force_close` would be the best to avoid unnecessary changes.

Reply:
done